sources/packages/web/src/views/aest/student/StudentDetails.vue

```diff
@@ -13,21 +13,15 @@
                 ? StudentRestrictionStatus.Restriction
                 : StudentRestrictionStatus.NoRestriction
             "
-            :label="
```

Comment:
why is this removed?

Reply:
Apparently there was never a property in `student-restriction-chip` called label and no usage found inside the component.

![image](https://user-images.githubusercontent.com/54600590/231289522-76d38e54-1678-4f70-8d78-199b67fa792f.png)